### PR TITLE
Recover password view

### PR DESCRIPTION
**Pull request by quichey**
-also leader of team can kick members from team

### CODE REVIEW

File: app/views/users/passwords/edit.html.haml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.content
```
Comment (ckvan) [inline app/views/users/passwords/edit.html.haml:1]: It's good that the file was made into a .haml

File: app/views/users/passwords/edit.html.haml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  = render :partial => 'users/nav_bar', :locals => { :current_user => current_user }
+
+.content
+  = render :partial => 'users/flash_messages', :locals => { :flash => flash }
+
+.container-fluid.session-container
+  .row
+    .col
+    .form-wrapper.col-md-4.col-10
```
Comment (ckvan) [inline app/views/users/passwords/edit.html.haml:10]: Formatting is important for polishing.

File: edit.html.haml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  = render :partial => 'users/nav_bar', :locals => { :current_user => current_user }
+
+.content
+  = render :partial => 'users/flash_messages', :locals => { :flash => flash }
+
+.container.title-container
+  %h2 Update Your Account
+
+
+-# .container.form-container
```
Comment (ckvan) [inline edit.html.haml:11]: Could delete this comment.

File: edit.html.haml [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+      .col-md-4.form-img-wrapper
+        = image_tag @user.image.url
+
+        %label.cabinet.btn.btn-primary
+          .text
+            Upload new image
+          = f.file_field :image, class: %w[file]
+
+
+        -# .img-upload.btn.btn-primary
```
Comment (ckvan) [inline edit.html.haml:27]: Is this for future use/extension? If not, I can delete it as part of maintenance.

File: app/views/users/_display_users.html.haml [hunk 5]
```diff
@@ -55,3 +55,6 @@
                 %div.container.title-container
                   = form_tag(promote_to_leader_path(team_id: @team.id, user_id: user.id), :method => 'PATCH', class: %w[form form-inline my-2 my-lg-0]) do
                     = submit_tag "Promote to Leader", name: nil, class: %w[button btn btn-outline-success my-2 my-sm-0], data: { confirm: "Are you sure?" }
+                %div.container.title-container
```
Comment (ckvan) [inline app/views/users/_display_users.html.haml:58]: Nice addition of privilege.

File: features/kick_from_team.feature [hunk 6]
```diff
@@ -0,0 +1,25 @@
+Feature: allow leaders of a team to kick members from the team
```
Comment (ckvan) [inline features/kick_from_team.feature:1]: These scenarios seem very similar to existing remove member scenarios. If the delete feature across different levels of hierarchy is not too big, you could condense into one file.

**Review by ckvan — APPROVED** [review]

File: edit.html.haml (hunk 4 above)
Comment (quichey) [reply edit.html.haml:27]: this file was a mistake. Accidentally hit paste while going through folders, didn't find what i pasted

**Pull request merged**